controllers/networking-calico/charts/networking-calico/values.yaml

```diff
@@ -9,19 +9,32 @@ resources: {}
   # - name: calico-node
   #   sourceRepository: github.com/projectcalico/calico
   #   repository: quay.io/calico/node
-  #   tag: v3.7.2
+  #   tag: v3.8.2
```

Comment:
Please remove all the concrete images + image tags here, that's double maintenance that we don't need. The image vector is the source of truth.

Reply:
you mean the tags? this section was added previously for the imageVectorOverwrite.

Reply:
I would not have something concrete in this image vector overwrite example. If somebody wants to overwrite it the exact values have to be looked up in the image vector anyways.